dev/breeze/tests/test_selective_checks.py

```diff
@@ -415,6 +417,42 @@ def assert_outputs_are_printed(expected_outputs: dict[str, str], stderr: str):
                 id="Only docs builds should run - no tests needed",
             )
         ),
+        (
+            pytest.param(
+                ("task_sdk/src/airflow/sdk/random.py",),
+                {
+                    "all-python-versions": "['3.9']",
+                    "all-python-versions-list-as-string": "3.9",
+                    "python-versions": "['3.9']",
+                    "python-versions-list-as-string": "3.9",
+                    "ci-image-build": "true",
+                    "prod-image-build": "false",
+                    "needs-api-tests": "false",
+                    "needs-helm-tests": "false",
+                    "run-kubernetes-tests": "false",
+                    "run-tests": "true",
+                    "run-task-sdk-tests": "true",
+                    # TODO: docs-build should be true once docs are added
```

Comment:
It's where `FileGroupForCi.DOC_FILES` is defined.

Maybe better to add it there actually @kaxil instead of leaving TODO. Little harm done this way, just docs building happening when only task_sdk files change - but we might avoid some false positives in the future.

Reply:
fixed in https://github.com/apache/airflow/pull/43185/commits/77c786d56f96ef88eef65ef6c0ba6d58811b8820